controllers/scopeinstance_controller.go

```diff
@@ -104,7 +101,9 @@ func (r *ScopeInstanceReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	}
 
 	// create required roleBindings and clusterRoleBindings.
+	log.Log.Error(nil, "XXX calling ensureBindings")
 	if err := r.ensureBindings(ctx, in, st); err != nil {
+		log.Log.Error(err, "XXX ensureBindings returned an error")
```

Comment:
I think it is good to do an error log here and remove the below `log.Log.Info()` call.

A tiny nit though. Since it is an Error log message I don't think we need to state that it has "returned an error". Users may also not be aware of what "ensureBindings" is so we should also probably go with a description that is more user-friendly. Maybe something like:
```suggestion
		log.Log.Error(err, "creating (Cluster)RoleBindings")
```

Reply:
@everettraven you are seeing my debug logging :) when I want to debug things I often will print them as ERRORs so I can guarantee they will show up and I prefix them with XXX so I can grep them easily :)

Reply:
I do think we should keep this one as an ERROR level log since we are communicating an error